<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerEditor</class>
 <widget class="QWidget" name="LayerEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="moveUpButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/arrow-up.png</normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/arrow-up.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/list-remove.png</normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/list-remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/list-add.png</normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/list-add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="moveDownButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/arrow-down.png</normaloff>../../../../../usr/share/icons/oxygen/base/16x16/actions/arrow-down.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListView" name="layerView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
